src/lib/styles/components/_sign-up.scss

```diff
@@ -1,8 +1,8 @@
-/********** 
- 22. Sing-up Section Styles
+/**********
+ 22. Sign-up Section Styles
 **********/
 
-.sing-up {
+.sign-up {
```

Comment:
Have the class referencing this style been updated already? Or wasn't actually using any style before?

Reply:
Yup! I had updated the classname previously, but not the CSS

Reply:
Cool.!